src/shared/constants/web3.constants.ts

```diff
@@ -31,6 +31,11 @@ export const CONNECTOR_LABELS: DynamicObject<string, EConnectorType, AllKeysRequ
   [EConnectorType.WALLET_CONNECT]: "Wallet Connect"
 };
 
+export const CONNECTOR_DESCRIPTIONS: DynamicObject<string, string, AllKeysRequired> = {
```

Comment:
Maybe we can merge CONNECT_ object into a single one now that there are multiple cases, something like CONNECTOR_METADATA